easybuild/easyblocks/n/numpy.py

```diff
@@ -166,6 +166,21 @@ def get_libs_for_mkl(varname):
         if fft:
             extrasiteconfig += "\n[fftw]\nlibraries = %s" % fft
 
+        suitesparseroot = get_software_root('SuiteSparse')
+        if suitesparseroot:
+            amddir = os.path.join(suitesparseroot, 'AMD')
+            umfpackdir = os.path.join(suitesparseroot, 'UMFPACK')
```

Comment:
In a remark that is now hidden, can't seem to find it myself anymore (maybe something went wrong with submitting it urgh).

Basically, I would make very sure that these directories are there:

```python
if not os.path.exists(amddir) or not os.path.exists(umfpackdir):
    raise EasyBuildError("Expected SuiteSparse subdirectories are not both there: %s, %s",
                         amddir, umfpackdir)